src/arc.ts

```diff
@@ -1,29 +1,54 @@
 import { Arc } from "@daostack/client";
+import Util from "lib/util";
 import { Observable } from "rxjs";
 
 const Web3 = require("web3");
 
+const INFURA_KEY = "16bDz7U53RbXysQiYOyc";
```

Comment:
This shouldn't go in the codebase, it should be read from an environment variable (you can put it in a .env file for development)

Reply:
I personally prefer to use a file containing configuration information, rather than environment variables.  I don't like that the app has to know anything so closely tied to the OS.  A config file can be structured to have env-specific sections, like "dev", "staging", "production" or "kovan", "ganache", "live", etc.  Such a config file can be maintained in the repo if desired.

Reply:
but we dont want to share our INFURA key publicly with the world, thats the whole point of using environment variables.

Reply:
I'd agree in general, but there is notthing secret about the infura key, it is sent in plain text with each request.